crates/extensions/c8y_mapper_ext/src/json.rs

```diff
@@ -74,7 +76,7 @@ mod tests {
 
         let entity = EntityMetadata::main_device("foo".to_string());
         let output =
-            from_thin_edge_json_with_timestamp(single_value_thin_edge_json, timestamp, &entity);
+            from_thin_edge_json_with_timestamp(single_value_thin_edge_json, timestamp, &entity, "");
```

Comment:
Not that happy to use an empty string. I feel better to use an `Option` here.

Reply:
In this specific case, `Option<String>` makes things a bit artificial.

- The c8y mapper will have to deal with `None` and `Some("")` as a non empty type string is mandatory.
- The measurement type received on the topic is actually a string, possibly empty.

Reply:
I agree, I will keep this as `string`.